libsplinter/src/peer/error.rs

```diff
@@ -14,11 +14,13 @@
 
 use std::{error, fmt};
 
+/// Represents the errors that could be raised by the `PeerManager`
 #[derive(Debug, PartialEq)]
 pub enum PeerManagerError {
+    /// `PeerManager` start up failed
     StartUpError(String),
+    /// Error raised when a message failed to send
     SendMessageError(String),
-    RetryEndpoints(String),
```

Comment:
This change is not represented by the commit message.

Reply:
added to commit